packages/web3-eth-contract/src/types.ts

```diff
@@ -88,26 +180,308 @@ export type PayableTxOptions = PayableCallOptions;
 
 export interface NonPayableMethodObject<Inputs = unknown[], Outputs = unknown[]> {
 	arguments: Inputs;
+	/**
+	 * Will call a method and execute its smart contract method in the EVM without sending any transaction. Note calling cannot alter the smart contract state.
```

Comment:
```suggestion
	 * This will call a method and execute its smart contract method in the EVM without sending any transaction. Note calling cannot alter the smart contract state.
```